_layouts/blog_by_tag.html

```diff
@@ -9,21 +9,55 @@
+  <!-- Category selector: desktop -->
+    <div class="container full desktop-only">
+        <div class="info-block blog-nav row">
+            <div class="col"><a href="{{ site.baseurl_root }}/blog">{% t blog.allposts %}</a></div>
+            <div class="col {% if filename == 'urgent' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/urgent.html">{% t blog.urgent %}</a></div>
+            <div class="col {% if filename == 'releases' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/releases.html">{% t blog.releases %}</a></div>
+            <div class="col {% if filename == 'community' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/community.html">{% t blog.community %}</a></div>
+            <div class="col {% if filename == 'dev diaries' %}checked{% endif %}"><a href="{{ site.baseurl_root }}/blog/tags/dev%20diaries.html">{% t blog.meetinglogs %}</a></div>
+        </div>
+    </div>
+  <!-- End category selector: desktop -->
+  <!-- Category selector: mobile -->
+  <div class="container full">
+      <div class="info-block row center-xs" id="pick-platform">
+          <div class="mob dropdowndrop">
+              <input id="filter" type="checkbox" name="category-filter"/>
+              <label for="filter">{% t blog.filter %}</label>
+              <ul id="menu">
+                  <li><a href="{{ site.baseurl_root }}/blog">{% t blog.allposts %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/urgent.html">{% t blog.urgent %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/releases.html">{% t blog.releases %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/community.html">{% t blog.community %}</a></li>
+                  <li><a href="{{ site.baseurl_root }}/blog/tags/dev%20diaries.html">{% t blog.meetinglogs %}</a></li>
+              </ul>
+          </div>
+      </div>
+  </div>
+  <!-- End category selector: mobile -->
+</div>
+
 <div class="site-wrap">
     <section class="container">
             <div class="row">
                 <!-- left two-thirds block-->
                 <div class="left two-thirds col-lg-8 col-md-8 col-sm-8 col-xs-12">
-                    <div class="info-block">
+                  <div class="info-block">
                         <div class="row center-xs">
-                        <div class="page-title">
-		                  <h2 class="inline"><span class="kicks">{{ tag.name }}</span></h2>
-		                </div>
-                    </div>
-                    
+                          <div class="page-title">
+                          {% if tag.name != "Dev Diaries" %}
+                            <h2 class="inline"><span class="kicks">{{ tag.name }}</span></h2>
+                          {% else %}
+                            <h2 class="inline"><span class="kicks">{% t blog.meetinglogs %}</span></h2>
+                          {% endif %}
+		                      </div>
```

Comment:
mixes spaces and tabs

Reply:
I still have to figure this out. The website is mixed. Some documents use spaces and some use tabs. Some pages are even mixed and they use both tabs and spaces. This file is a good example of my confusion.
I often refrain to change indentation to avoid useless diffs, but i think we should decide for either using tabs or spaces and adapt the files we edit accordingly.

In this case i might have just made confusion, because i see i changed indentation. I will just use tabs here, but the problem remains.

Reply:
I would not change existing code, just try to not mix it when adding new stuff

Reply:
My point is that "not change existing code" and "not mix it when adding new stuff" is not possible. Whatever i do (without reindenting everything), it will be mixed. The part i added (the top) uses tab, but the bottom uses mixed tabs and spaces.

My additions at the bottom are mixed because whatever i choose to do i have to change indentation of other parts, which i wanted to avoid. Do you suggest to reindent all the parts i touch?

Reply:
Not really that important but the lines beneath and above are all spaces, so I would just change this line to all spaces.

Reply:
Alright. i'll do that.

Reply:
Oddly enough, the indentation looks good on VS code, but looks off as shown here with another editor. i will have to check some settings i guess.

Reply:
Github probably uses a different visual tab width.